amazon-kinesis-client/src/test/java/software/amazon/kinesis/leases/HierarchicalShardSyncerTest.java

```diff
@@ -205,6 +205,66 @@ public void testCheckAndCreateLeasesForShardsIfMissingAtLatest() throws Exceptio
 
     }
 
+    @Test
```

Comment:
Comment for file in general: Can we add comments explaining what each test is doing, and explicitly calling out any changes with code comments to common setup between very similar tests.

Reply:
Yep, addressing